projects/observability/src/shared/dashboard/data/graphql/metric-aggregation/metric-aggregation-data-source.model.ts

```diff
@@ -42,7 +42,7 @@ export class MetricAggregationDataSourceModel extends GraphQlDataSourceModel<Met
 
   public getData(): Observable<MetricAggregation> {
     const spec = new ExploreSpecificationBuilder().exploreSpecificationForKey(
-      this.metric.metric,
+      this.metric.metric as string,
```

Comment:
This typecast is the only change in this file. What changed that it's needed now?

Reply:
`exploreSpecificationForKey` expects first parameter to be a string and this.metric.metric can be either a string or `AttributeExpression`.  For this data source metric.metric will be a string hence typecasting is required. Cannot directly assign it as string since `metric` is of type `ExploreSelectionSpecificationModel` which is a general model used in other data sources as well